grafana-plugin/src/components/ScheduleQuality/ScheduleQuality.styles.ts

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getScheduleQualityStyles = (_theme: GrafanaTheme2) => {
+  return {
+    root: css`
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      gap: 5px;
+    `,
+
+    quality: css`
+      line-height: 16px;
+    `,
+
+    tag: css`
+      font-size: 12px;
+      padding: 5px 10px;
+
+      &--danger {
+        // TODO: emotionjs
+        background-color: var(--tag-background-danger);
```

Comment:
This needs to be sorted out with @raphael-batte as these tag colors don't really exist in grafana and we need to get rid of the whole custom vars file